### PR TITLE
data: khalidx3

### PR DESCRIPTION
**Pull request by khalidx3**
Added my profile
Check List (Check all the applicable boxes)
 The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2584"></a>

### CODE REVIEW

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

File: data/khalidx3.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/khalidansariX3",
+      "icon": "FaTwitter"
+    }
+  {
+      "name": "Follow me on Github",
+      "url": "https://github.com/khalidx3",
+      "icon": "FaGithub"
+    }
```
Comment (eddiejaoude) [inline data/khalidx3.json:18]: Missing json comma

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Invalid json, missing commas

**Review by khalidx3 — COMMENTED** [review]
re-reviewed

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**